tests/test_memcache.py

```diff
@@ -166,6 +166,13 @@ def test_disconnect_all_delete_multi(self):
         ret = self.mc.delete_multi({'keyhere': 'a', 'keythere': 'b'})
         self.assertEqual(ret, 1)
 
+    @mock.patch.object(_Host, 'readline')
+    def test_memcached_error_unexpected_reply(self, mock_readline):
+        """Test that the debug log executes properly in error response."""
+        mock_readline.return_value = 'SET'
+        cache_key = ':cache:key:1'
```

Comment:
Is there a special significance to this cache key name? Also, it looks like it doesn't need to be an intermediate variable.

Can you add an assertion for the debug output?

Reply:
You are such an ass; these are stylistic decisions in writing the test.  The test passes because the debug log doesn't blow up, and fails if you take out the change because of the python 3 TypeError.   I'm done making your changes -- if you want to fix this library then just take my work and do it yourself because you are too damn picky.   I've already converted our apps away from using this library, and was trying to make it better for people that still use it.

Reply:
I'm simply trying to improve the quality of the tests. Please don't assume malicious intent. I'll see if I can address my suggestions if you're not interested.

Reply:
Well I personally like when there are intermediate variables in tests because it both names the value, and allows a debugger to introspect things.   It also makes it easier later on if you want to then assert things about the key, because most likely if its not a named variable, another dev extending the test will copy the magic value all over the place.   I've seen this time and time again.

Reply:
I guess the problem with your suggestions is that its nit-picking a test that I choose to add.   At least my test mocks out communication with Memcahed server to a certain extent.   How about spending the time on the existing tests,  mocking them, and adding more test coverage rather than be upset about an intermediate variable and a lack of assertion in the test case.   The test runner  still fails a test if something raises exception, so I feel like these are subjective stylistic preferences and not something enforced by flake8 or that actually improve the test.   I wasn't willing to assert anything about the logging because I think its ridiculous it doesn't even use the logging module, but I think I've stated that already.

Reply:
I just saw the passive aggressive thumbs up about adding an assertion of the awful logging methodology in your code base @linsomniac , but I am not going to do that.